Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -560,11 +560,14 @@ void AssignSceneID()
         //    build indices if there are disabled scenes in build settings, and
         //    if no scene is in build settings then Editor and Build have
         //    different indices too (Editor=0, Build=-1)
+        // -> use lower case path because BuildPipeline.BuildPlayer can give
+        //    different case than is used in the editor
         // => ONLY USE THIS FROM POSTPROCESSSCENE!
         public void SetSceneIdSceneHashPartInternal()
         {
             // get deterministic scene hash
-            uint pathHash = (uint)gameObject.scene.path.GetStableHashCode();
+            string scenePath = gameObject.scene.path.ToLower();
```

Comment:
great fix. could you add exactly the PR description as comment here please?
the two lines won't be obvious anymore in 5-10 years :)

Reply:
done